tests/evaluator-tests.js

```diff
@@ -168,8 +168,8 @@ describe('Evaluator', () => {
       var currentProjectGetter = [
         ['projects'],
         ['session', 'currentProjectId'],
-        (projects, currentProjectId) => {
```

Comment:
We can change rules and fix the respective lint issues in future PRs as we change our minds about styling rules. :+1: